.github/workflows/ci-lint.yaml

```diff
@@ -9,6 +9,14 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           version: v1.31
+  bash-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run Shellcheck
+        uses: azohra/shell-linter@v0.4.0
+        with:
+          severity: "error"
```

Comment:
Using error rather than a more sensitive value. It gives warnings for odd things like certain files that aren't bash ("Dockerfile is not scanned, if its bash start with a shebang..." is a warning) and other things.

Error has caught all the real issues that I've seen (quoting, looping, spacing, etc)